crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -162,25 +189,44 @@ impl ParsedQProperty {
                     ))
                 }
 
-                if let Some(read) = read {
+                let cxx_name_str = if let Some(name) = cxx_name {
+                    name.to_string()
+                } else {
+                    ident.to_string().to_case(Case::Camel)
```

Comment:
i think the idea was to camelCase only if there was no rust_name or cxx_name ? Rather than if there is no cxx_name. Or have i misremembered ?

(but then all of this will start changing soon anyway)

Reply:
So it might become more like this maybe?
```rust
let name = if cxx_name.is_none() && rust_name.is_none() {
   Name::new(ident)
     .with_cxx_name(/* to camel */)
} else {
  Name::new(ident)
    /* if there is a cxx_name */
    .with_cxx_name(...)
   /* if there is a rust_name */
    .with_rust_name(...)

  // or can this just be the Name::from_ident_and_attrs(...) ?
};
```